test/unit/gateways/pay_hub_test.rb

```diff
@@ -0,0 +1,423 @@
+require 'test_helper'
+
+class PayHubTest < Test::Unit::TestCase
+  def setup
+    @gateway = PayHubGateway.new(add_credentials)
+    @credit_card = credit_card('371449635398431', credit_card_params)
+    @amount = 200
+    @options = optional_params
+  end
+
+  def test_successful_purchase
+    @gateway.expects(:ssl_request).returns(successful_purchase_or_authorize_response)
+
+    response = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success response
+    assert response.test?
+    assert_match(/^SUCCESS/, response.message.to_s)
```

Comment:
You're using `assert_equal 'SUCCESS', response.message` in the remote tests, any particular reason to use a regexp in these ones?